source/connection-string/connection-string-spec.rst

```diff
@@ -180,9 +180,9 @@ The values in connection options MUST be URL decoded by the parser. The values c
 - Strings:
     The value
 - Integer:
-    The value parsed as a integer
+  The value parsed as a integer. If the value is the empty string, then a WARN level logging message MUST be issued and the key MUST be ignored.
```

Comment:
This warrants bumping the spec version and adding a changelog entry. Since this is a functional change an introducing new tests, I think a minor version bump is appropriate.

Reply:
I have bumped the minor version and added a changelog entry.